tests/functions.sh

```diff
@@ -537,8 +537,11 @@ function approve_pull_request () {
             exit 1
         fi
     fi
-    real_title=$(echo $all_prs | jq -r --arg pr_title "$pr_title" 'select(.[].title | startswith($pr_title)) | .[].title' | head -n 1)
+    real_title=$(echo $all_prs | jq -r --arg pr_title "$pr_title" '.[] | select(.title | startswith($pr_title)) | .title' | head -n 1)
```

Comment:
@edaena This looks like it's just an optimization, correct? The result should be the same

Reply:
@andrebriggs It doesn't work for me locally if I leave it the other way. startswith() returns results that don't start with the specified string.